### DISCUSSION TITLE
installazione docker

### DISCUSSION THREAD

**Discussion by nunzio87** [indomus/forum#1911] [GENERALE - Discussioni generiche]
ciao, sto avendo problemi con doker su raspberry.

ho seguito alla lettera le istruzioni per installare un sistema pulito di Raspberry Pi OS

poi questa guida https://indomus.it/guide/come-installare-docker-su-raspberry-pi-con-raspbian/

e poi questa https://indomus.it/guide/come-installare-e-configurare-home-assistant-core-con-docker-su-un-raspberry-pi-gia-in-uso/

e mi blocco, arrivo al punto (Tramite comando “docker run”) e non vado oltre, aspetto di poter fare il primo accesso ma non funziona.

è evidente che qualcosa che non va c'è di sicuro ma non capisco dove.

**Comment by snakuzzo** [comment]
Se non dai evidenza con qualcosa... Difficile aiutarti
Se fai questo che ti dice?

```
docker logs home-assistant
```

**Reply by nunzio87** [reply]
scusa ma non sapevo nemmeno come aiutarvi ad aiutarmi, comunque mi restituisce questo
 ```
 pi@raspberrypi:~ $ docker logs home-assistant
s6-overlay-suexec: fatal: can only run as pid 1
s6-overlay-suexec: fatal: can only run as pid 1
s6-overlay-suexec: fatal: can only run as pid 1
s6-overlay-suexec: fatal: can only run as pid 1
```
una serie molto lunga, ma sono tutti così

**Reply by snakuzzo** [reply]
Devi omettere `--init` dal comando `docker run` (dobbiamo aggiornare la guida).

Fai così...rimuovi il container appena creato e poi ridai il comando `docker run`

```
docker container rm home-assistant
docker run -d --name="home-assistant" -e "TZ=Europe/Rome" -v "/home/pi/homeassistant:/config" -v "/run/dbus:/run/dbus:ro" --net=host --restart always ghcr.io/home-assistant/home-assistant:stable
```

(ovviamente sostituendo i dati dei volumi, porte, etc con quelli che servono a te)

**Reply by nunzio87** [reply]
perdonami, ma oviamente mica tanto...   dati dei volumi, porte, etc non so quali siano `pi@raspberrypi:~ $ docker container rm home-assistant
Error response from daemon: You cannot remove a restarting container 81d7dc6415239de48d93b19faa082bf184cbadcdbc539e11d65b8f39fb49c2c9. Stop the container before attempting removal or force remove
pi@raspberrypi:~ $ docker container rm home-assistant -f
home-assistant` 
ho aggiunto -f ad intuito. i dati dei volumi ecc ti chiedo di illuminarmi

**Reply by snakuzzo** [reply]
Se hai seguito con copia-incolla la guida, una volta rimosso il container, fai conto che per ricrearlo devi  rilanciare il comando `docker run` senza `--init` ... tutto qua

**Reply by nunzio87** [reply]
ah ok, grazie mille.

**Reply by snakuzzo** [reply]
Intanto è stata aggiornata la guida correggendo il comando `docker run`

**Reply by nunzio87** [reply]
ora funziona infatti
